cmd/kube-aws-approver/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/golang/glog"
+
+	awsapprover "github.com/coreos/kubecsr/pkg/approver/aws"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+)
+
+var (
+	kubeconfig  = flag.String("kubeconfig", "", "kubeconfig file with acces to cluster. (testing only)")
+	regionName  = flag.String("region-name", "", "When empty uses metadata service to extract.")
+	allowedASGs = flag.String("allowed-asgs", "", "Comma separated string of allowed ASGs")
+)
+
+func main() {
+	flag.Set("logtostderr", "true")
```

Comment:
What's this for?

Reply:
https://godoc.org/github.com/golang/glog 
```
-logtostderr=false
	Logs are written to standard error instead of to files.
```

Reply:
also bootkube uses this https://github.com/kubernetes-incubator/bootkube/blob/506fed7ade32929fda481a6ed2a4a3d6820e6daa/pkg/util/log.go#L14